### DISCUSSION TITLE
Galaxy Brain badge

### DISCUSSION THREAD

**Discussion by shahzod222** [community/community#19587] [Other Feature Feedback, Questions, & Ideas] [answered]
How do I get a Galaxy Brain badge if I have answered 20 discussions?
![Снимок экрана (103)](https://user-images.githubusercontent.com/93871956/176101912-1f60de37-2ba2-4909-8361-0827a39d83bf.png)

**Comment by D7EAD** [comment]
Hi!

I believe discussions answered before the release of the achievement may not count. If this is indeed the case and I am not mistaken, you would need to simply answer two additional discussions to acquire it!

You can learn more about achievements and their tiers [here](https://github.com/Schweinepriester/github-profile-achievements).

**Reply by shahzod222** [reply]
Today, I have already answered 3 discussions, but unfortunately the badge didn't appear.

**Reply by D7EAD** [reply]
@shahzod222, you mean to tell me you have received three **Marked as Answer**'s today, and have yet to receive the badge? How odd!

Try disabling badges and re-enabling them on your profile. I have no idea if this would even make a difference, but hey, worth a try! If that doesn't work, then this sounds particularly buggy to me and may require help from someone more familiar with this, such as a Github employee.

**Reply by shahzod222** [reply]
I have tried to reload achievements, but no, it doesn't work.

**Comment by melvincwng** [comment, marked as answer]
@shahzod222, I have addressed the same question or issue in this discussion thread here #18293.

Can you please take a look at the marked answer over there?

Basically, you CANNOT mark your own answers to your own questions. Doing so will cause a bug that prevents the badge from showing in your profile.

Hope this clarifies.

**Reply by melvincwng** [reply]
@shahzod222, if my answer was helpful, I would appreciate it if you could kindly do the following:

1. 'Mark as answer'
2. Follow me on GitHub

Am trying to build my network of connections and friends on GitHub ^^

Otherwise, take care and have a good day ahead 😄

**Reply by shahzod222** [reply]
@melvincwng  Don't you know how fix that bug which prevents the badge from showing in my profile?

**Reply by melvincwng** [reply]
@shahzod222, to fix that bug:
1. Uncheck all your self-marked answers.
2. Have another user mark one of your answers again.
3. The badge should automatically appear.

In short for that badge to appear, you absolutely cannot have your own marked answers.

Hope this is helpful, if it is, please 'mark as answer' and consider following me on GitHub. Thanks!

**Reply by shahzod222** [reply]
Thank you very much. I have earned Galaxy Brain badge!!!!

**Reply by melvincwng** [reply]
@shahzod222, awesome enjoy your new shiny badge ^^ 🎉 !

Thanks for marking as answer, stay in touch in the future ^^

Do tag me or follow me on GitHub if you have other questions about achievements in the future. I will be glad to assist 😄 

Take care, my friend

**Comment by ryudongjae** [comment]
Below is a compiled knowledge of what I know, after hours of scouring through discussion forums, and trying to understand how the mechanics work (for some of the achievements at least).

YOLO - Merge a PR directly without review (can add a friend as a reviewer). Repo should be public.

Pull Shark - Merge at least 2 PRs into a public repo.

Galaxy Brain:

Answer at least 2 discussions.
Take note, you CANNOT mark your own answers to your own questions.
They have to be marked by another user. Discussions should be in a public repository.
Also if you mark answers to your own questions, there could be a bug whereby the achievement won't show despite having >=2 answers marked by other users.
Reference:

I've answered 2 discussions but they don't gave me the Galaxy Brain badge. https://github.com/orgs/github-community/discussions/18454

Also, please provide a couple of hours (or maybe up to a day) for your new achievements to show. If they haven't shown by then, perhaps you did not hit some "hidden" criteria. In most cases, developers seem to get their achievements instantly though.
Contact 'Github Support' if you face any issues.

The community can feel free to chime in, and provide their own observations on how the requirements of the new achievements, in case I may have missed some stuff out.

Hope this answer has helped you, and all the best in getting the achievement! :)

**Reply by shahzod222** [reply]
Thank you for your suggestion! I've already earned Galaxy Brain badge. I hope you will get that badge too.

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Pavel-Shcherbo** [comment]
How do I get a Galaxy Brain badge if I have answered 5 discussions?

**Comment by Hazrat-Ali9** [comment]
How do I get a Galaxy Brain badge
Response
